docs/en/build.md

```diff
@@ -9,7 +9,14 @@ NodeMCU "application developers" just need a ready-made firmware. There's a [clo
 Occasional NodeMCU firmware hackers don't need full control over the complete tool chain. They might not want to setup a Linux VM with the build environment. Docker to the rescue. Give [Docker NodeMCU build](https://hub.docker.com/r/marcelstoer/nodemcu-build/) a try.
 
 ### Linux Build Environment
-NodeMCU firmware developers commit or contribute to the project on GitHub and might want to build their own full fledged build environment with the complete tool chain. There is a [post in the esp8266.com Wiki](http://www.esp8266.com/wiki/doku.php?id=toolchain#how_to_setup_a_vm_to_host_your_toolchain) that describes this.
+NodeMCU firmware developers commit or contribute to the project on GitHub and might want to build their own full fledged build environment with the complete tool chain. The NodeMCU project includes a ready-made tool chain for Linux/x86-64 by default. After working through the [Build Options](#build-options) below, simply start the build process with
```

Comment:
Nit pick: we don't really _include_ it (anymore) but we download one on the fly if necessary. Maybe link to https://github.com/nodemcu/nodemcu-firmware/blob/dev/Makefile#L225 here?